File: app/src/main/java/com/chooloo/www/callmanager/service/CallService.java
package com.chooloo.www.callmanager.service;

import android.content.Intent;
import android.telecom.Call;
import android.telecom.InCallService;

import com.chooloo.www.callmanager.ui.call.OngoingCallActivity;

public class CallService extends InCallService {

    @Override
    public void onCallAdded(Call call) {
        super.onCallAdded(call);
        CallManager.sCall = call;
        Intent intent = new Intent(this, OngoingCallActivity.class);
        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        startActivity(intent);
    }

    @Override
    public void onCallRemoved(Call call) {
        super.onCallRemoved(call);
        CallManager.sCall = null;
    }

}

File: app/src/main/java/com/chooloo/www/callmanager/service/CallService.kt
package com.chooloo.www.callmanager.service

import android.content.Intent
import android.telecom.Call
import android.telecom.InCallService
import com.chooloo.www.callmanager.ui.call.CallActivity

class CallService : InCallService() {
    override fun onCallAdded(call: Call) {
        super.onCallAdded(call)
        CallManager.sCall = call
        goToCallActivity()
    }

    override fun onCallRemoved(call: Call) {
        super.onCallRemoved(call)
        CallManager.sCall = null
    }

    private fun goToCallActivity() {
        val intent = Intent(this, CallActivity::class.java)
        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
        startActivity(intent)
    }
}